src/chart/graph/Thumbnail.ts

```diff
@@ -0,0 +1,204 @@
+
+    group = new graphic.Group();
+    _parent: graphic.Group;
+
+    _widthProportion: number;
+    _heightProportion: number;
+
+    _selectedRect: graphic.Rect;
+    _wrapper: graphic.Rect;
+    _layoutParams: LayoutParams;
+
+    _treeModel: GraphSeriesModel;
+
+    constructor(containerGroup: graphic.Group) {
+        containerGroup.add(this.group);
+        this._parent = containerGroup;
+    }
+
+    render(seriesModel: GraphSeriesModel, api: ExtensionAPI) {
+        const model = seriesModel.getModel('thumbnail');
+        const group = this.group;
+
+        group.removeAll();
+        if (!model.get('show')) {
+          return;
+        }
+
+        this._treeModel = seriesModel;
+
+        const childrenNodes = (this._parent.children()[0] as graphic.Group).children();
+        const symbolNodes = (childrenNodes[0] as graphic.Group).children();
+        const lineNodes = (childrenNodes[1] as graphic.Group).children();
+
+        const backgroundColor = model.getModel('backgroundColor');
+        const borderColor = model.getModel('borderColor');
+        const thumbnailWidth = model.get('width');
+        const thumbnailHeight = model.get('height');
+        const selectedDataBackground = model.getModel('selectedDataBackground');
+
+        this._layoutParams = {
+            pos: {
+                left: model.get('left'),
+                right: model.get('right'),
+                top: model.get('top'),
+                bottom: model.get('bottom')
+            },
+            box: {
+                width: api.getWidth(),
+                height: api.getHeight()
+            }
+        };
+
+          const layoutParams = this._layoutParams;
+
+          this._widthProportion = <number>thumbnailWidth / layoutParams.box.width;
+          this._heightProportion = <number>thumbnailHeight / layoutParams.box.height;
+
+          const thumbnailGroup = new graphic.Group();
+
+        for (const node of symbolNodes) {
+              const sub = (node as graphic.Group).children()[0];
+              const x = (node as SymbolClz).x;
+              const y = (node as SymbolClz).y;
+
+              const subThumbnail = new (sub as any).constructor({
+                  shape: {
+                        ...(sub as graphic.Path).shape,
+                        width: 5,
+                        height: 5,
+                        x: (x * 0.25 - 2.5),
+                        y: (y * 0.25 - 2.5)
+                      },
+                  style: (sub as graphic.Path).style,
+                  z: 3,
+                  z2: 151
+              });
+
+              thumbnailGroup.add(subThumbnail);
+        }
+
+      for (const node of lineNodes) {
+          const line = (node as graphic.Group).children()[0];
+          const lineThumbnail = new ECLinePath({
+              style: (line as ECLinePath).style,
+              shape: (line as ECLinePath).shape,
+              scaleX: 0.25,
+              scaleY: 0.25,
+              z: 3,
```

Comment:
`z` will be force set in https://github.com/pe-3/echarts-funnel-newfeat/blob/a236499bcfea9b990462b7138e7d3357d3590c8b/src/core/echarts.ts#L2373

Reply:
sorry, I don't understand how to use `z` in this link

Reply:
Just remove it